samples/chatbot-easy-rag-kotlin/src/main/resources/META-INF/resources/components/demo-title.js

```diff
@@ -0,0 +1,72 @@
+import {LitElement, html, css} from 'lit';
+
+export class DemoTitle extends LitElement {
+
+    static styles = css`
+      h1 {
+        font-family: "Red Hat Mono", monospace;
+        font-size: 60px;
+        font-style: normal;
+        font-variant: normal;
+        font-weight: 700;
+        line-height: 26.4px;
+        color: var(--main-highlight-text-color);
+      }
+
+      .title {
+        text-align: center;
+        padding: 1em;
+        background: var(--main-bg-color);
+      }
+
+      .explanation {
+        margin-left: auto;
+        margin-right: auto;
+        width: 50%;
+        text-align: justify;
+        font-size: 20px;
+      }
+
+      .explanation img {
+        max-width: 60%;
+        display: block;
+        float: left;
+        margin-right: 2em;
+        margin-top: 1em;
+      }
+    `
+
+    render() {
+        return html`
+            <div class="title">
+                <h1>Financial Assistant</h1>
+            </div>
+            <div class="explanation">
+                This demo shows how to build a chat bot powered by LLM and retrieval-augmented generation (RAG).
+                The application uses the Easy RAG extension to retrieve information about financial products from provided
+                documents stored in a catalog directory.
+                The description of the different accounts is ingested into a Redis database and relevant information
+              is sent to the LLM before answering the user.It also provides stock price information for selected companies.
```

Comment:
Could you add a few example prompts on this page, to recommend what the user should try? We generally do this in other similar sample apps, and I find it extremely helpful. Otherwise, the user who runs the app might not have clear ideas what to say to the bot.

For example
- "You are stupid" to trigger the moderation. Also tell the user to look at http://localhost:8080/q/dev-ui/io.quarkiverse.mailpit.quarkus-mailpit/mailpit-ui afterwards

Reply:
- "What is the current stock price of GOOG?" (Probably also make an explicit note that it does not really use live data, but random numbers, in case somebody doesn't read the code)

Reply:
- "What is the minimum opening deposit for the standard savings account?"